pyproject.toml

```diff
@@ -46,6 +47,7 @@ marshmallow-sqlalchemy = "0.24.2"
 werkzeug = "^0.16.0"
 html2text = "^2020.1.16"
 pycryptodome = "3.9.8"
+Flask-Migrate = "^3.1.0"
```

Comment:
Is flask-migrate used anywhere?

Reply:
No, it's not anymore. I switched directions. I'll go ahead an remove it.